### ISSUE TITLE
showing validation samples greater than training samples

### ISSUE THREAD

**Issue by vinayakumarr** [tflearn/tflearn#107]
I am able to run imdb program successfully. But always while running a program shows validaton samples greater than training samples

**Comment by aymericdamien** [comment]
What are you using for validation_set argument?

**Comment by vinayakumarr** [comment]
//for splitting

X_train, X_test, y_train, y_test = train_test_split(X, y,test_size=0.2,random_state=42)

print("Preprocessing")
X_train1 = X_train.values.T.tolist()
X_test1 = X_test.values.tolist()
y_train1 = y_train.values.T.tolist()
y_test1 = y_test.values.tolist()

# Data preprocessing
# Sequence padding
trainX = pad_sequences(X_train1, maxlen=200, value=0.)
testX = pad_sequences(X_test1, maxlen=200, value=0.)

# Converting labels to binary vectors
trainY = to_categorical(y_train, nb_classes=0)
testY = to_categorical(y_test, nb_classes=0)

//model fitting

model.fit(trainX, trainY, validation_set=(testX, testY), show_metric=True, batch_size=128,n_epoc=100)
